site/content/integrate/admin-guide/admin-message-attachments/_index.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Can we simplify the file name to `site/content/integrate/admin-guide/message-attachments/_index.md`

Reply:
That's a good idea.